server/index.js

```diff
@@ -124,6 +124,26 @@ app.get('/api/statuses', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.post('/api/tasks/create', (req, res, next) => {
+  const { userId, taskName, statusId, description, dateCreated, deadline, projectId } = req.body;
+  if (!userId || !taskName || !statusId || !description || !dateCreated || !deadline) {
+    throw new ClientError(400, 'User ID, task name, status ID, description, date created and deadline are required');
+  }
+
+  const sql = `
+  insert into "tasks" ("taskId", "statusId", "userId", "projectId", "taskName", "description", "dateCreated", "deadline")
+               values (default, $1, $2, $3, $4, $5, $6, $7);
+  `;
+
+  const params = [statusId, userId, projectId, taskName, description, dateCreated, deadline];
+
+  db.query(sql, params)
+    .then(result => {
+      res.status(201).json(result.rows);
```

Comment:
Generally, the client you should receive the thing they created, in case they want to do something with it. So be sure to
1. include a `returning` clause in your `insert` statement
2. send back the inserted record instead of an array